src/promo/MessageTile.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Box from '@material-ui/core/Box'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+import Button from '../Button'
+import Typography from '../Typography'
+import neutralPalette from '../styles/palettes/neutral'
+
+const styles = theme => ({
+  body: {
+    marginBottom: 16
+  },
+  box: {
```

Comment:
Are these repeatable attributes? All measures will be divisible by 4 so it's great if we can consolidate the spacing somehow.

Reply:
If our baseline spacing unit is 4 then we should really configure that in the theme. Currently our baseline spacing unit is the default (8 pixels).

Changing to 4 may mess with a lot of the default components and third party libraries though, so we'll want to be careful when changing it. May not be an issue though - I haven't tried it.

Either way we should aim to use the theme spacing methods wherever possible instead of magic numbers.

Reply:
I don't think we should change the spacing unit from 8px to 4px; MUI specifically recommends using a 8px spacing https://material-ui.com/customization/spacing/

I agree with Nick that it'll probably quite negatively affect other components.

But yes, I'll change it to not have magic numbers around. We can always do `theme.spacing(0.5)` for 4px.

Reply:
Ah yes - Let's keep to 8.